src/scripts.js

```diff
@@ -0,0 +1,72 @@
+ loadJSON(function(response) {
```

Comment:
YOu can use the created() function on the Vue instance itself to load the data, so everything thing else can initialise while you wait.

Like this 😄 https://codepen.io/mikestreety/pen/jLYGbW